packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Server/EnvVars/elements.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import EnvIcon from 'react-icons/lib/go/key';
+import Input from '@codesandbox/common/lib/components/Input';
+
+export const EnvironmentIcon = styled(EnvIcon)`
+  margin-right: 0.6rem;
+  font-size: 1rem;
+  margin-left: 4px;
+`;
+
+export const IconWrapper = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const InputContainer = styled.div`
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  margin-bottom: 0.5rem;
+`;
+
+export const ErrorMessage = styled.div`
+  color: ${props => props.theme.red};
+  font-size: 12px;
+  margin: 0.5rem 0;
+  font-style: italic;
+`;
+
+export const EnvName = styled(Input)`
+  margin-left: 0;
+  margin-right: 0;
+  margin-bottom: 0.25rem;
```

Comment:
I don't know why, but it seems this margin got lost somewhere?

![image](https://user-images.githubusercontent.com/587016/64970339-dcf8b100-d8a5-11e9-8e33-3fe5c4927014.png)

Reply:
It needs to overwrite another one so its back to inline